### PR TITLE
feat: project credit batches perf

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-registry#1757

-  [add getClassesByIssuerQuery](https://github.com/regen-network/regen-web/commit/59ea665b4e7c223bd9f7d4bb04b3c70f18f6a0dc)
- [add getProjectsByAdminQuery](https://github.com/regen-network/regen-web/commit/681bf55ffe686764a46b14bfb723bac6ce080a00)
- use them in `Dashboard component`

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1. https://deploy-preview-2041--regen-marketplace.netlify.app/ecocredits/portfolio

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie you can check that the `projects` and `credit batches` tabs are displayed faster now.

File: web-marketplace/src/graphql/indexer/ClassesByIssuer.graphql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+query IndexerClassesByIssuer($issuer: String) {
+  allClassIssuers(condition: { latest: true, issuer: $issuer }) {
+    nodes {
+      blockHeight
```
Comment (ryanchristo) [inline web-marketplace/src/graphql/indexer/ClassesByIssuer.graphql:4]: Do we need block height?

File: web-marketplace/src/graphql/indexer/IssuersByClassId.graphql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+query IndexerIssuersByClassId($classId: String) {
+  allClassIssuers(condition: { latest: true, classId: $classId }) {
+    nodes {
+      blockHeight
```
Comment (ryanchristo) [inline web-marketplace/src/graphql/indexer/IssuersByClassId.graphql:4]: Same here, it looks like this is unused. Any reason for including it?

**Review by ryanchristo — APPROVED** [review]
tACK. Huge performance improvement on my end. Nice work!

**Comment by ryanchristo** [comment]
It looks like this also resolves an issue with fetching existing projects. I'm seeing projects on the deploy preview that I'm not able to see in the staging environment (even after waiting for all requests to finish).

File: web-marketplace/src/graphql/indexer/ClassesByIssuer.graphql (hunk 1 above)
Comment (flagrede) [reply web-marketplace/src/graphql/indexer/ClassesByIssuer.graphql:4]: Not really indeed, I will remove it.

File: web-marketplace/src/graphql/indexer/IssuersByClassId.graphql (hunk 2 above)
Comment (flagrede) [reply web-marketplace/src/graphql/indexer/IssuersByClassId.graphql:4]: Same https://github.com/regen-network/regen-web/pull/2041#discussion_r1303876078

**Comment by erikalogie** [comment]
looks great!

**Pull request merged**